backend/src/app/exceptions/projects.py

```diff
@@ -8,3 +8,7 @@ class FailedToAddProjectRoleException(Exception):
     """
     Exception raised when a project_role can't be added for some reason
     """
+
+
+class NoneStrictPostiveNumberOfSlots(Exception):
```

Comment:
(This should also be changed everywhere where the exception is used, not just in here)
```suggestion
class NoStrictlyPositiveNumberOfSlots(Exception):
```